cmd/eib/main.go

```diff
@@ -1,7 +1,72 @@
 package main
 
-import "fmt"
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+const (
+	argConfigFile = "config"
+	argVerbose    = "verbose"
+)
+
+func init() {
+	log.SetFormatter(&log.TextFormatter{
+		FullTimestamp:    true,
+		QuoteEmptyFields: true,
+	})
+	log.SetOutput(os.Stdout)
+}
+
+func processArgs() (*config.ImageConfig, error) {
+	var (
+		configFile string
+		verbose    bool
+	)
+
+	flag.StringVar(&configFile, argConfigFile, "", "full path to the image configuration file")
+	flag.BoolVar(&verbose, argVerbose, false, "enables extra logging information")
+	flag.Parse()
+
+	handleVerbose(verbose)
+	imageConfig, err := handleImageConfig(configFile)
+
+	return imageConfig, err
+}
+
+func handleVerbose(verbose bool) {
+	if verbose {
+		log.SetLevel(log.DebugLevel)
+	}
+}
+
+func handleImageConfig(configFile string) (*config.ImageConfig, error) {
+	if configFile == "" {
+		return nil, fmt.Errorf("the \"%s\" argument must be specified", argConfigFile)
```

Comment:
Just FYI, we should avoid embedding such data in the errors and use structured logging instead but this can be done once we replace logrus.